osu.Game/Screens/Edit/Timing/TapTimingControl.cs

```diff
@@ -192,13 +210,24 @@ private void reset()
             editorClock.Seek(selectedGroup.Value.Time);
         }
 
+        private List<HitObject> hitObjectsInTimingRange(EditorBeatmap beatmap, ControlPointGroup selectedGroup)
+        {
+            // If the first group, we grab all hitobjects prior to the next, if the last group, we grab all remaining hitobjects
+            double firstGroupTime = beatmap.ControlPointInfo.Groups.Any(x => x.ControlPoints.Any(y => y is TimingControlPoint) && x.Time < selectedGroup.Time) ? selectedGroup.Time : double.MinValue;
+            double nextGroupTime = beatmap.ControlPointInfo.Groups.FirstOrDefault(x => x.ControlPoints.Any(y => y is TimingControlPoint) && x.Time > selectedGroup.Time)?.Time ?? double.MaxValue;
```

Comment:
- please use `startTime` / `endTime` as variable names here
- please use `beatmap.ControlPointInfo.TimingPoints` immediately reather than going through linq `Any()` to select them